db/trie/trie_test.go

```diff
@@ -390,47 +365,28 @@ func TestInsert(t *testing.T) {
 	require.Equal(testV[2], b)
 
 	// delete "cat"
-	logger.Info().Msg("Del[cat]")
+	t.Log("Del[cat]")
 	err = tr.Delete(cat)
 	require.Nil(err)
-	require.Equal(EmptyBranchNodeHash, tr.RootHash())
+	require.True(tr.isEmptyRootHash(tr.RootHash()))
 	require.Nil(tr.Stop(context.Background()))
 }
 
 func TestBatchCommit(t *testing.T) {
 	require := require.New(t)
-	testutil.CleanupPath(t, testTriePath)
-	defer testutil.CleanupPath(t, testTriePath)
 
-	tr, err := NewTrie(db.NewBoltDB(testTriePath, cfg), "test", KeyLengthOption(8))
+	tr, err := NewTrie(KeyLengthOption(8))
 	require.Nil(err)
 	require.Nil(tr.Start(context.Background()))
+	trieDB := tr.DB()
 	// insert 3 entries
 	require.Nil(tr.Upsert(cat, testV[2]))
 	require.Nil(tr.Upsert(car, testV[1]))
 	require.Nil(tr.Upsert(egg, testV[4]))
-	require.Nil(tr.Commit())
 	c, _ := tr.Get(cat)
 	require.Equal(testV[2], c)
-	root := tr.RootHash()
```

Comment:
why remove them?

Reply:
trie doesn't have the concept of "commit"

Reply:
i don't quite get it, doesn't trie have Commit() method?

this is to test close the trie/db w/o calling Commit(), then all pending changes from last Commit should not be persisted to trie

Reply:
that's what db should worry about. Our trie is based on kv store. Given a root hash in someway, trie is able to construct the data structure with the data stored in kv store. Thus, whether or not the data has been written to db shouldn't be something Trie concerns.